qt/python/mantidqt/widgets/algorithmprogress/dialog_widget.py

```diff
@@ -67,6 +70,16 @@ def __init__(self, parent, model):
         self.presenter = AlgorithmProgressDialogPresenter(self, model)
         self.presenter.update_gui()
 
+    def closeEvent(self, *args):
```

Comment:
The argument should be `event` of type `QCloseEvent` according to the documentation http://doc.qt.io/qt-5/qwidget.html#closeEvent